jb/itch5/ut_compute_book_cache_aware.cpp

```diff
@@ -0,0 +1,410 @@
+#include <jb/itch5/compute_book_cache_aware.hpp>
```

Comment:
Please remove this file from the current PR, we can introduce it in the next one.